bin/templates/cordova/lib/builders/ProjectBuilder.js

```diff
@@ -16,34 +16,33 @@
        specific language governing permissions and limitations
        under the License.
 */
-
-var Q = require('q');
-var fs = require('fs');
-var util = require('util');
-var path = require('path');
-var shell = require('shelljs');
-var spawn = require('cordova-common').superspawn.spawn;
-var events = require('cordova-common').events;
-var CordovaError = require('cordova-common').CordovaError;
-var check_reqs = require('../check_reqs');
-
-var GenericBuilder = require('./GenericBuilder');
-
-var MARKER = 'YOUR CHANGES WILL BE ERASED!';
-var SIGNING_PROPERTIES = '-signing.properties';
-var TEMPLATE =
+/* eslint no-self-assign: 0 */
+/* eslint no-unused-vars: 0 */
+
+let Q = require('q');
+let fs = require('fs');
+let path = require('path');
+let shell = require('shelljs');
+let spawn = require('cordova-common').superspawn.spawn;
+let events = require('cordova-common').events;
+let CordovaError = require('cordova-common').CordovaError;
+let check_reqs = require('../check_reqs');
```

Comment:
Why can't we use const here too? Or at least leave them using var for now.

Reply:
> Using const means that we can't spy on them unfortunately.

I don't get why that would be the case : thinking:

> I can revert them to var if you think that would be better.

I'd prefer that, since it reduces the amount of unrelated changes.